scripts/mkcloud

```diff
@@ -1431,11 +1431,19 @@ function expand_steps()
             if [ $onecmd = $cmd ] ; then
                 found=1
                 case "$cmd" in
+                    setupcompute|instcompute)
+                        corrected=${cmd/compute/nodes}
+                        echo "WARNING: the '$cmd' step is deprecated; use '$corrected' instead." >&2
```

Comment:
this might go unnoticed... a deadly complain 123 "..." would force people to see + correct it.